packages/babylon/src/tokenizer/index.js

```diff
@@ -467,6 +467,7 @@ export default class Tokenizer extends LocationParser {
       const assign =
         this.input.charCodeAt(this.state.pos + 2) === charCodes.equalsTo;
       if (assign) {
+        // $FlowFixMe
```

Comment:
Flow doesn't like that the inheritance chain goes:

`BaseParser (which has hasPlugin) -> CommentsParser -> LocationParser -> Tokenizer -> UtilParser (which has expectPlugin)`